### ISSUE TITLE
[draft] TWE: Create Template Issue: [name of template]

### ISSUE THREAD

**Issue by bradyse** [hackforla/internship#329]
### Overview
We need to create a template issue for [replace with name of template issue] so that we have consistent, transparent, defined and self-supporting structure for people creating them.

### Action Items
- [ ] Add to the Project Board
- [ ] Add the milestone: Project Management
- [ ] Add a feature label specific to what the template is related to, if available.  If not clear what you should use, ask product.
- [ ] Any place you see `[replace...]` replace both the text and the brackets with what it is asking for
- [ ] Create a draft of new template by copying the content block below to a comment on this issue, which will be known as our Draft
- [ ] In your draft, Fill out the top part (called front matter) with the info it needs.  See the Resources section below for definition and example.
- [ ] Define any Dependencies (or remove the dependency section)
- [ ] Customize the Overview section
- [ ] Fill out the Action Items section (in your draft) by defining the steps that are required to be done, keeping the following in mind:
   - [ ] The person working on this issue might be new to the org, so the issue should be able to be worked on without any prior knowledge
   - [ ] If the person needs to add links, or put documents in specific locations, the place where they can do that action must be linked in the resources section
      - [ ] the links provided by the issue creator should be in the same order as the Action Items 
- [ ] If documents or folders (sheets, docs, presentations, dashboard, etc.) will be created, find the document template that should be used by checking the WIKI Templates by role pages.  
  - [ ] if there is a template
     - [ ] Add a link to the template into the ` ### Resources`  section of your draft
  - [ ] if there is no template yet
    - [ ] put a placeholder into the` ### Resources`  section of the draft `[TWE: Template: [replace with name of type]]`
- [ ] Get sign-off by product on the draft
- [ ] Create the template issue on the repo
- [ ] Update the appropriate place in wiki for your template (see resources section)

#### WIKI Templates
- by role pages
  - [TWE: WIKI: UX Research Guides Templates](https://github.com/hackforla/internship/wiki/UX-Research-Guides-Templates) 
  - [Design]
  - [Product]
  - [Engineering]
- default document templates, in case you don't have one yet can found on the [Design Process Overview page](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

#### Content to copy for draft
```
Draft of [replace with name of type] template issue
---
name: TWE: Create Template Issue: [replace with name of template]
about: 
title: TWE: [replace with action verb] [replace with name of template]
labels: 'feature: board / google drive / tools', 'feature: missing', 'size: missing', 'role: missing', 'milestone: missing'
assignees: 
---
# Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- [ ] Find WIKI page for this issue by looking at role pages [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team) - then add the appropriate link to the resources section
- [ ] Find document template that should be used [Templates & Product Branding](https://github.com/hackforla/internship/wiki/Design-Process-Overview) - then add that specific template link into the resources section
- [ ] If no dependencies, delete dependency section
- [ ] any place you see `[replace...]` replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] add appropriate labels and remove the corresponding missing labels
- [ ] delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
WE NEED TO DO X FOR Y REASON

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, etc.

### Resources
```
### Resources
<details>
<summary>Front Matter Fields and Meaning</summary>

- name: (name that appear on the new issue tab)
- about: (is the description that appears next to the issue template on the new issue tab
- title: (the title that appears when the issue is created.  User then customizes this title for their issue)
- labels: (choose the labels that will be the same each time, and choose missing labels for the ones that depend on content)
- assignees: 
</details>

<details>
<summary>Front Matter Field example</summary>

- name: 'Research: Roadmap'
- about: The start of a new roadmap
- title: 'TWE: Research Roadmap [name of type]'
- labels: 'feature: research, feature: roadmap, milestone: missing, role: UI/UX research,
-   size: 0.50pt'
- assignees: ''
</details>

**Comment by bradyse** [comment]
Draft of `TWE: Create Slide Deck for Presentation of Findings` template issue
---
name: TWE: Create Template Issue: Create Slide Deck for Presentation of Findings
about: 
title: TWE: RP[###]: Create Slide Deck for Presentation of Findings
labels: 'feature: board / google drive / tools', 'feature: missing', 'size: missing', 'role: missing', 'milestone: missing'
assignees: 
---
# Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- [ ] Find WIKI page for this issue by looking at role pages [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team) - then add the appropriate link to the resources section
- [ ] Find document template that should be used [Templates & Product Branding](https://github.com/hackforla/internship/wiki/Design-Process-Overview) - then add that specific template link into the resources section
- [ ] If no dependencies, delete dependency section
- [ ] any place you see `[replace...]` replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] add appropriate labels and remove the corresponding missing labels
- [ ] delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
WE NEED TO DO X FOR Y REASON

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, etc.

### Resources

**Comment by Aparna1Gopal** [comment]
Draft of `TWE: Create Slide Deck for Presentation of Findings` template issue
---
name: TWE: Create Template Issue: Create Slide Deck for Presentation of Findings
about: 
title: TWE: RP[###]: Create Slide Deck for Presentation of Findings
labels: 'feature: board / google drive / tools', 'feature: missing', 'size: missing', 'role: missing', 'milestone: missing'
assignees: 
---
# Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- [ ] Find WIKI page for this issue by looking at role pages [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team) - then add the appropriate link to the resources section
- [ ] Find document template that should be used [Templates & Product Branding](https://github.com/hackforla/internship/wiki/Design-Process-Overview) - then add that specific template link into the resources section
- [ ] If no dependencies, delete dependency section
- [ ] any place you see `[replace...]` replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] add appropriate labels and remove the corresponding missing labels
- [ ] delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

**Comment by Aparna1Gopal** [comment]
**Notes: To discuss during with one-on-one with UXR Lead** 

- Discuss the correct labels to give this issue: 'feature: board / google drive / tools', 'feature: missing', 'size: missing', 'role: missing', 'milestone: missing'
'feature: research', 'feature: missing', 'size: missing', 'role: missing', 'milestone: missing'

**Comment by Aparna1Gopal** [comment]
[feature: board / google drive / tools]
[feature: presentation templates]
[feature: research]
All issues involving research
[role: UX research lead]
[size: 1pt]

**Comment by Aparna1Gopal** [comment]
1. Progress: Met 1-1 with UXR Lead Sara B on November 07, 2022 (5PM PST-6:30PM PST)
2. Blockers: Discussed items on this list
https://github.com/hackforla/internship/issues/329#issuecomment-1305209593
5. Availability: 2-3 hours 
6. ETA: As per feedback from UXR Lead, will re-work on the sections and then message UXR Lead to confirm completion lunch that they are able to treat this issue as per requirements in the next stage. 
7. Pictures or links (if necessary):

_* remember to add links to the top of the issue if they are going to be needed again._

**Comment by Aparna1Gopal** [comment]
@bradyse I have completed this issue and it can be reviewed. Thanks! 

I have completed this issue. Could you please review [[draft] TWE: Create Template Issue: Create Slide Deck for Presentation of Findings](https://github.com/hackforla/internship/issues/329#top)
#329 and let me know if any further changes need to be made? Thanks!

**Comment by Aparna1Gopal** [comment]
1. Progress: Met 1-1 with UXR Lead Sara B on November 21, 2022 (5PM -6:30PM, Pacific Time)
2. Blockers: None. Discussed feedback from previous week regarding 'Dependencies' and relevant 'Research Plan (RP)' and specific issues required for Data Analysis.
3. Availability: 2-3 hours
4. ETA: The UXR Lead, Sara B will be reviewing this issue. Once done, I will be retesting this template issue. 
Also discussed the issue relevant to data analysis: https://github.com/hackforla/internship/issues/330
6. Pictures or links (if necessary):

_* remember to add links to the top of the issue if they are going to be needed again._

**Comment by Aparna1Gopal** [comment]
**Notes:** 
@Aparna1Gopal 

Relevant Resources: 

- [2022: Data Analysis - TWE: Research: Guide Creation] (https://github.com/hackforla/internship/issues/330)TWE: Research: Guide Creation: Interview Expert (Linda) #330
- [2021: Example of a completed Data Analysis: RP001](https://github.com/hackforla/internship/wiki/Research-Plan-1:-IS21-Interns)

**Comment by bradyse** [comment]
@Aparna1Gopal Please copy this [comment ](https://github.com/hackforla/internship/issues/329#issuecomment-1289836751)into a new comment below to test it out. 

The only thing that I didn't get to was creating a specific presentation slide template. For the time being, just use the generic template.

**Comment by ExperimentsInHonesty** [comment]
Need to test the template in this [comment](https://github.com/hackforla/internship/issues/329#issuecomment-1289836751).  And give feedback about what is not clear.

**Comment by cbutcher318** [comment]
currently testing in #352

**Comment by pandanista** [comment]
> Finished testing in #352
> Stopped at section "Review Research Plan Documentation" as I was confused how to move forward

@cbutcher318 I have reviewed the testing in #352, and noticed a couple of things:
<img width="743" alt="Screenshot 2023-01-31 at 4 47 22 PM" src="https://user-images.githubusercontent.com/57029070/215917927-dc843464-6633-4d53-9f36-cd1deed97392.png">
In Resources 2.02, RP000 should be RP017
Resources 2.04 should be 2.03 

Your comment is noted. I believe adding [RP 000] to the existing instructions will help clarify your question. So, instead of "Review wiki page", we can modify it to "Review RP[Replace 000] wiki page".
<img width="848" alt="Screenshot 2023-01-31 at 4 48 52 PM" src="https://user-images.githubusercontent.com/57029070/215917904-446dab74-4cb9-412a-a79d-3a462dbbc4f7.png">

Instead of "Review the Research Plan Google Doc", we can modify it to "Review RP[Replace 000] Google Doc"; and include a section to "Customize Resource for this Research Plan" under the "Customize Resource for this Research Plan's Google Drive Folder" and include the URL to the "RP[Replace 000] Google Doc" in the resources section for easy access and reference.  
<img width="848" alt="Screenshot 2023-01-31 at 5 03 53 PM" src="https://user-images.githubusercontent.com/57029070/215919834-1fbccb22-a2d9-476d-8afc-dce7aeaa2003.png">

The part that mentions "write questions into comments", I recommend providing a step by step guide to show assignee how they could make comments on a Google Doc in case they are confused. 

In your case, since you were testing RP017, you can review RP017's wiki page and research plan Google Doc. Leave comments on the Google Doc, and then follow the suggested deck structure in the template to make the deck and add content into it. However RP017 is mostly blank, it might be easier to test using RP001 since it is more complete.

**Comment by cbutcher318** [comment]
@pandanista here are some additional comments:
* on "Review RP[Replace 000] wiki page", what is the purpose? What do you want them to review and take away from the review?
* on "Review the Research Plan Google Doc", is there an exemplar for them to compare to? It might be useful to give list of starting questions (eg., "As you read through the doc, pay attention to: Is the information clear and complete? Is there any piece of information that seems to be missing? Could the writing be revised to make it more clear?" etc.) Otherwise a person might not know what they are supposed to be paying attention to and commenting/questioning on.
    * could they use "suggestion" mode instead of or along with comments?

- the "Create the slide deck" section is also vague. Instructions need to be more granular, telling them to use the template 1.03, etc.
- BUT ALSO, we need a new research slide template. The current slidedeck template is for a meeting (unless this slide is supposed to be a meeting slidedeck with breaks, etc.).
- I suggest a research slide exemplar be created and linked to in resources

**Comment by priyatalwar** [comment]
@priyatalwar please look at it with Bonnie

**Comment by pandanista** [comment]
@bradyse Would you update us on the status of this issue?

**Comment by bonniewolfe** [comment]
Hi @bradyse !
I have reviewed this issue and found the following items have not yet been resolved.  Are you able to look at these comments and made the changed needed to the template?

- Review notes from Christian https://github.com/hackforla/internship/issues/329#issuecomment-1404053812
- Review notes from Yingie https://github.com/hackforla/internship/issues/329#issuecomment-1411311380
- Follow up notes from Christian https://github.com/hackforla/internship/issues/329#issuecomment-1432194507

**Comment by bradyse** [comment]
> Hi @bradyse ! I have reviewed this issue and found the following items have not yet been resolved. Are you able to look at these comments and made the changed needed to the template?
> 
> * Review notes from Christian [TWE: Template Issue: Research: Presentation Slide Deck: Create #329 (comment)](https://github.com/hackforla/internship/issues/329#issuecomment-1404053812)
> * Review notes from Yingie [TWE: Template Issue: Research: Presentation Slide Deck: Create #329 (comment)](https://github.com/hackforla/internship/issues/329#issuecomment-1411311380)
> * Follow up notes from Christian [TWE: Template Issue: Research: Presentation Slide Deck: Create #329 (comment)](https://github.com/hackforla/internship/issues/329#issuecomment-1432194507)

@pandanista 
I have reviewed these notes and made edits to the [main template draft](https://github.com/hackforla/internship/issues/329#issuecomment-1289836751). I believe that Christian had issue with the RP he was working from because it had no Google Doc Research Plan (but now it does! 😅). That being said, I made edits according to your feedback and added a couple more steps, reordering the numbers, etc. I believe it is ready for another round of testing. It may be helpful to use RP017 once we can finalize the data analysis.

**Comment by sunannie27** [comment]
@bradyse Is this for the research lead or for someone else? If it is for the research lead, please add `ready for: research lead`.

**Comment by pandanista** [comment]
@bradyse Thank you very much for revising the template. I have created a new issue (#475) to test this template once the data analysis is done.

**Comment by ExperimentsInHonesty** [comment]
Ying and I are going to 
- [customize this presentation template](https://docs.google.com/presentation/d/1_toVSyxXPlNAbvP7jP7hve_TOn2xqSpZFaUppi_QpGs/edit#slide=id.g135a9586b34_1_6) for research findings
- customize the related issues so that they can use it.

**Comment by ExperimentsInHonesty** [comment]
I think this should get worked on by Ying and Jinyan  (see prior research presentations) and then review with product if you have specific questions.

Here is an example of a presentation from TDM Research Team
[TDM Calculator Round 3 User Testing Insights](https://drive.google.com/file/d/195OfQTz8budjo92RydHnwar743fWwPqf/view)

**Comment by jinyan0425** [comment]
@pandanista Ying, what is your thought on the [draft](https://github.com/hackforla/internship/issues/329#issuecomment-1289836751)? Shall we tweak the template a bit to be more consistent with other templates that were recently deployed and then test it?

**Comment by pandanista** [comment]
@jinyan0425: Yes, we need to tweak it and then test it with any research plan after we are done with the data analysis stage. I took another look at this template after our meeting, and here are the next steps:
- Update [the first comment](https://github.com/hackforla/internship/issues/329#issue-1421656893) with the latest create issue template
- Update the [TWE: Slide Deck Template](https://docs.google.com/presentation/d/1m7cYJ9jeH-z6bUv9xiHOjxmBUmANZrKs7YtQbNVfqgY/template/preview)
- Update the presentation template in [this comment](https://github.com/hackforla/internship/issues/329#issuecomment-1289836751)

Like we mentioned yesterday, let's prioritize the RQA related templates since they are more urgent. We can work on this afterwards.

**Comment by ExperimentsInHonesty** [comment]
@jinyan0425 Please clean this issue up and make any new issues that follow this, and prioritize them according to the above comment https://github.com/hackforla/internship/issues/329#issuecomment-2359371182.

**Comment by sunannie27** [comment]
@jinyan0425 Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by jinyan0425** [comment]
@sunannie27 
1. We have not started it yet, we are prioritizing the RQA issues. We will work on this issue after 624 is done. Does that sound good to Product?
- https://github.com/hackforla/internship/issues/618
- https://github.com/hackforla/internship/issues/620
- https://github.com/hackforla/internship/issues/621
- https://github.com/hackforla/internship/issues/624

2. NA.
3. I will be on vacation for the first three weeks of October, and may barely have time working on this.
4. Not sure at this moment.
5. NA.
